### PR TITLE
added TA Assignment test

### PR DESCRIPTION
**Pull request by azewudie**

### CODE REVIEW

File: TAInformation/views.py [hunk 1]
```diff
@@ -154,78 +163,78 @@ def get(self, request):
     # Postcondition: Creates new user, if data entered validates successfully and user doesn’t already exist.
     # Side Effects: Message indicating result is displayed at the bottom of the “Create Courses” form
     def post(self, request):
-        # Extract data from form
-        match request.session['role']:
-            case AccountType.ADMIN.value:
-                current_user = UserAdmin(
+         Extract data from form
```
Comment (bptome) [inline TAInformation/views.py:166]: should "extract data from form" be a comment here?

**Review by bptome — CHANGES_REQUESTED** [review]

File: TAInformation/views.py [hunk 2]
```diff
@@ -154,78 +163,78 @@ def get(self, request):
...
+             case _:  # Enforcement of selection should never allow this case to be reached
+                 new_user: BaseUser
 
         result_dict = current_user.create_user(new_user)
 
         return render(request, "create_user.html", {'result': result_dict['result'], 'message': result_dict['message']})
 
 
-# this is a dummy method. will eventually use user_id and return a User() class of the user that matches the user_id
+     this is a dummy method. will eventually use user_id and return a User() class of the user that matches the user_id
```
Comment (OmarHHammad) [inline TAInformation/views.py:237]: this should be a comment

File: virtualenv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Collecting virtualenv
```
Comment (OmarHHammad) [inline virtualenv:1]: this file can be deleted

**Review by OmarHHammad — COMMENTED** [review]
Overall, LGTM! Some minor tweaks but nice job

**Review by bptome — APPROVED** [review]

**Review by vaishnavi-manikonda — APPROVED** [review]
LGTM

**Pull request merged**